File: cfgs/java_cfgs_foo/FIND_MINIMUM_DIFFERENCE_PAIR_1.java
Nodes
1;special;;;
2;statement;fun_1(foo_1);14;
3;statement;int diff = Integer.MAX_VALUE;15;
4;statement;int i = 0;16;
5;conditional;foo_1 < (foo_2 - 1);17;
6;conditional;(foo_1[foo_2 + 1] - foo_3[foo_4]) < foo_5;18;
7;statement;diff = foo_1[foo_2 + 1] - foo_3[foo_4];18;
8;statement;i++;18;
9;statement;return foo_1;19;
10;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;9;
5;7;8;
6;8;
7;5;
8;10;
9;

File: cfgs/pythoncfgs/FIND_MINIMUM_DIFFERENCE_PAIR_1.py
Nodes
1;statement;foo_1 = fun_1(foo_2);8;
2;statement;foo_1 = 10 ** 20;9;
3;conditional;fun_1(foo_1 - 1);10;
4;statement;return foo_1;13;
5;conditional;foo_1[foo_2 + 1] - foo_3[foo_4] < foo_5;11;
6;statement;foo_1 = foo_2[foo_3 + 1] - foo_4[foo_5];12;
7;exit;;
Edges
0;2;
1;3;
2;4;5;
3;7;
4;3;6;
5;3;
6;
